sktime/datatypes/_utilities.py

```diff
@@ -392,7 +392,15 @@ def update_data(X, X_new=None):
             return np.concatenate([X, X_new], axis=2)
     #  if y is pandas, we use combine_first to update
     elif isinstance(X_new, (pd.Series, pd.DataFrame)) and len(X_new) > 0:
-        return X_new.combine_first(X)
+        # using .combine_first() can mess with df frequencies, e.g. change 24H to 1D,
```

Comment:
no change request, but just want to point out another instance of `pandas` madness to @benHeid and @yarnabrina.

Related to "multiples of frequencies" failures and support.

Reply:
FYI @ciaran-g, I requested the test because things like this, in `pandas` tend to change without warning and then everything breaks... see the discussions around 2.2.0.